### PR TITLE
Make tests more deterministic

### LINKED ISSUE #17045: [FEA] Make cudf tests deterministic
**Is your feature request related to a problem? Please describe.**
Currently various cudf tests use random data generated by e.g. numpy.random. These data are not consistently seeded, meaning that each run could end up with slightly different information. In some cases this could result in sporadic errors due to numerical issues and edge cases. This is particularly problematic if those failures occur in CI and force us to rerun tests.

**Describe the solution you'd like**
We should update cudf tests to either specify a seed or stop using random number generation altogether in tests. The may be made on a case-by-case basis.

### PR DESCRIPTION
**Pull request by galipremsagar**
## Description

Fixes #17045

This PR removes randomness in our pytests and switches from using `np.random.seed` to `np.random.default_rng` in all of the codebase.

## Checklist
- [x] I am familiar with the [Contributing Guidelines](https://github.com/rapidsai/cudf/blob/HEAD/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.

### CODE REVIEW

**Comment by mroeschke** [comment]
If interested, I included this pre-commit check in pandas to check that `default_rng` usage is always seeded (admittedly someone could still pass `seed=None` to get the unseeded behavior)

https://github.com/pandas-dev/pandas/blob/main/.pre-commit-config.yaml#L210-L211

**Comment by vyasr** [comment]
I like the idea of a pygrep pre-commit check. We could also look out for `np.random.seed()`.

File: python/cudf/cudf/utils/hash_vocab_utils.py [hunk 1]
```diff
@@ -245,7 +245,7 @@ def hash_vocab(
     """
     Write the vocab vocabulary hashtable to the output_path
     """
-    np.random.seed(1243342)
```
Comment (galipremsagar) [inline python/cudf/cudf/utils/hash_vocab_utils.py:248]: @VibhuJawa Switching to `default_rng` changes the random values generated and thus updates to `python/cudf/cudf/tests/data/subword_tokenizer_data/bert_base_cased_sampled/vocab-hash.txt` have been made, will that be an issue for tokenizer?

Comment (VibhuJawa) [reply python/cudf/cudf/utils/hash_vocab_utils.py:248]: I dont think it should be , will have to test with some common vocabulary files to be sure

**Review by charlesbluca — CHANGES_REQUESTED** [review]
Looks like there are still two remaining appearance of `np.random.seed` in notebooks/performance-comparisons/performance-comparisons.ipynb and docs/cudf/source/user_guide/performance-comparisons/performance-comparisons.ipynb - should those also get updated?

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -95,6 +95,18 @@ repos:
         entry: 'pytest\.xfail'
         language: pygrep
         types: [python]
+      - id: no-unseeded-default-rng
```
[the remark this answers is not in the document]
Comment (wence-) [reply .pre-commit-config.yaml:98]: FWIW, I quite like the separate entries, since the name of the entry gives some information as to what went wrong. If the regex gets complicated, I find it hard to see what's going on. But I don't have strong feelings here.

File: python/cudf/cudf/testing/dataset_generator.py [hunk 3]
```diff
@@ -48,16 +48,19 @@ def __init__(
...
         self.cardinality = cardinality
         self.null_frequency = null_frequency
-        self.generator = generator
+        if generator is None:
+            self.generator = lambda: [
+                np.random.default_rng(seed=0).integers(0, 100)
+                for _ in range(100)
+            ]
+        else:
+            self.generator = generator
```
Comment (wence-) [inline python/cudf/cudf/testing/dataset_generator.py:63]: This seems to be changing the default generator?

File: python/cudf/cudf/tests/test_binops.py [hunk 4]
```diff
@@ -574,7 +588,6 @@ def test_series_cmpop_mixed_dtype(cmpop, lhs_dtype, rhs_dtype, obj_class):
 )
 def test_series_reflected_ops_scalar(func, dtype, obj_class):
     # create random series
-    np.random.seed(12)
     random_series = utils.gen_rand(dtype, 100, low=10)
```
Comment (wence-) [inline python/cudf/cudf/tests/test_binops.py:591]: Do we not need to send a seed into `gen_rand`?

File: python/cudf/cudf/tests/test_binops.py [hunk 5]
```diff
@@ -631,7 +644,6 @@ def test_series_reflected_ops_cudf_scalar(funcs, dtype, obj_class):
     cpu_func, gpu_func = funcs
 
     # create random series
-    np.random.seed(12)
     random_series = utils.gen_rand(dtype, 100, low=10)
```
Comment (wence-) [inline python/cudf/cudf/tests/test_binops.py:647]: Same question here about a missing seed?

File: python/cudf/cudf/tests/test_categorical.py [hunk 6]
```diff
@@ -252,10 +252,10 @@ def test_cat_series_binop_error():
 @pytest.mark.parametrize("num_elements", [10, 100, 1000])
 def test_categorical_unique(num_elements):
     # create categorical series
-    np.random.seed(12)
+    rng = np.random.default_rng(seed=0)
```
Comment (wence-) [inline python/cudf/cudf/tests/test_categorical.py:255]: I suppose it doesn't really matter, but why the change of seed?

File: python/cudf/cudf/tests/test_column.py [hunk 7]
```diff
@@ -31,12 +31,13 @@
 @pytest.fixture(params=dtypes, ids=dtypes)
 def pandas_input(request):
     dtype = request.param
-    rng = np.random.default_rng()
+    rng = np.random.default_rng(seed=None)
```
Comment (wence-) [inline python/cudf/cudf/tests/test_column.py:34]: I know this is a no-op change, but is there a reason you're not setting the seed for this rng?

File: python/cudf/cudf/tests/test_dataframe.py [hunk 8]
```diff
@@ -63,6 +63,9 @@
     pytest_xfail = pytest.mark.skipif
 
 
+rng = np.random.default_rng(seed=0)
+
+
```
Comment (wence-) [inline python/cudf/cudf/tests/test_dataframe.py:68]: *nit*: Would prefer not to have a global rng in this test file.

File: python/cudf/cudf/tests/test_dataframe.py [hunk 9]
```diff
@@ -428,7 +431,7 @@ def test_series_init_none():
 
 
 def test_dataframe_basic():
-    np.random.seed(0)
+    rng = np.random.default_rng(seed=0)
```
Comment (wence-) [inline python/cudf/cudf/tests/test_dataframe.py:434]: Having set a global `rng`, why produce a local one here?

File: python/cudf/cudf/tests/test_dataframe.py [hunk 10]
```diff
@@ -2844,8 +2848,7 @@ def test_cuda_array_interface(dtype):
 @pytest.mark.parametrize("data_type", dtypes)
 def test_from_arrow_chunked_arrays(nelem, nchunks, data_type):
     np_list_data = [
-        np.random.randint(0, 100, nelem).astype(data_type)
-        for i in range(nchunks)
+        rng.integers(0, 100, nelem).astype(data_type) for i in range(nchunks)
     ]
```
Comment (wence-) [inline python/cudf/cudf/tests/test_dataframe.py:2852]: *nit*: Make an `rng` here, rather than relying on the global one?

File: python/cudf/cudf/tests/test_dataframe.py [hunk 11]
```diff
@@ -3894,13 +3900,13 @@ def test_select_dtype_datetime_with_frequency():
 
 
 def test_dataframe_describe_exclude():
-    np.random.seed(12)
+    rng = np.random.default_rng(seed=0)
```
Comment (wence-) [inline python/cudf/cudf/tests/test_dataframe.py:3903]: Change of seed?

File: python/cudf/cudf/tests/test_dataframe.py [hunk 12]
```diff
@@ -3910,13 +3916,13 @@ def test_dataframe_describe_exclude():
 
 
 def test_dataframe_describe_include():
-    np.random.seed(12)
+    rng = np.random.default_rng(seed=0)
```
Comment (wence-) [inline python/cudf/cudf/tests/test_dataframe.py:3919]: Change of seed?

File: python/cudf/cudf/tests/test_dataframe.py [hunk 13]
```diff
@@ -3925,12 +3931,12 @@ def test_dataframe_describe_include():
 
 
 def test_dataframe_describe_default():
-    np.random.seed(12)
+    rng = np.random.default_rng(seed=0)
```
Comment (wence-) [inline python/cudf/cudf/tests/test_dataframe.py:3934]: Change of seed?

File: python/cudf/cudf/tests/test_dataframe.py [hunk 14]
```diff
@@ -3939,14 +3945,14 @@ def test_dataframe_describe_default():
 
 
 def test_series_describe_include_all():
-    np.random.seed(12)
+    rng = np.random.default_rng(seed=0)
```
Comment (wence-) [inline python/cudf/cudf/tests/test_dataframe.py:3948]: Change of seed?

File: python/cudf/cudf/tests/test_dataframe.py [hunk 15]
```diff
@@ -3962,13 +3968,13 @@ def test_series_describe_include_all():
 
 
 def test_dataframe_describe_percentiles():
-    np.random.seed(12)
+    rng = np.random.default_rng(seed=0)
```
Comment (wence-) [inline python/cudf/cudf/tests/test_dataframe.py:3971]: Change of seed?

File: python/cudf/cudf/tests/test_groupby.py [hunk 16]
```diff
@@ -1313,9 +1323,9 @@ def test_empty_groupby(func):
 
 
 def test_groupby_unsupported_columns():
-    np.random.seed(12)
+    rng = np.random.default_rng(seed=0)
```
Comment (wence-) [inline python/cudf/cudf/tests/test_groupby.py:1326]: Change of seed?

File: python/cudf/cudf/tests/test_index.py [hunk 17]
```diff
@@ -2645,8 +2645,10 @@ def test_isin_multiindex(data, values, level, err):
...
 
 
+rng = np.random.default_rng(seed=0)
+
 range_data = [
-    range(np.random.randint(0, 100)),
+    range(rng.integers(0, 100)),
     range(9, 12, 2),
     range(20, 30),
     range(100, 1000, 10),
```
Comment (wence-) [inline python/cudf/cudf/tests/test_index.py:2654]: TODO: Can we open an issue to turn this usage into a fixture, so that we don't do this at test collection time?

File: python/cudf/cudf/tests/test_index.py [hunk 18]
```diff
@@ -2831,7 +2833,7 @@ def test_rangeindex_append_return_rangeindex():
 
 
 index_data = [
-    range(np.random.randint(0, 100)),
+    range(rng.integers(0, 100)),
```
Comment (wence-) [inline python/cudf/cudf/tests/test_index.py:2836]: TODO: Can we open an issue to turn this usage into a fixture, so that we don't do this at test collection time?

File: python/cudf/cudf/tests/test_orc.py [hunk 19]
```diff
@@ -681,7 +681,6 @@ def test_orc_write_statistics(tmpdir, datadir, nrows, stats_freq):
 def test_orc_chunked_write_statistics(tmpdir, datadir, nrows, stats_freq):
     from pyarrow import orc
 
-    np.random.seed(0)
     supported_stat_types = supported_numpy_dtypes + ["str"]
```
Comment (wence-) [inline python/cudf/cudf/tests/test_orc.py:684]: *question*: Does `gen_rand_series` now arrange for a seed to be set?

File: python/cudf/cudf/tests/test_orc.py [hunk 20]
```diff
@@ -988,7 +985,7 @@ def test_orc_string_stream_offset_issue():
 
 def generate_list_struct_buff(size=100_000):
     rd = random.Random(1)
-    np.random.seed(seed=1)
+    rng = np.random.default_rng(seed=0)
```
Comment (wence-) [inline python/cudf/cudf/tests/test_orc.py:989]: Change of seed?

File: python/cudf/cudf/tests/test_orc.py [hunk 21]
```diff
@@ -1135,7 +1132,7 @@ def gen_map_buff(size):
     from pyarrow import orc
 
     rd = random.Random(1)
-    np.random.seed(seed=1)
+    rng = np.random.default_rng(seed=0)
```
Comment (wence-) [inline python/cudf/cudf/tests/test_orc.py:1135]: Change of seed?

File: python/cudf/cudf/tests/test_rank.py [hunk 22]
```diff
@@ -125,9 +125,11 @@ def test_rank_error_arguments(pdf):
     )
 
 
+rng = np.random.default_rng(seed=0)
+
 sort_group_args = [
     np.full((3,), np.nan),
-    100 * np.random.random(10),
+    100 * rng.random(10),
```
Comment (wence-) [inline python/cudf/cudf/tests/test_rank.py:132]: TODO: Can we open an issue to turn this usage into a fixture, so that we don't do this at test collection time?

File: python/cudf/cudf/tests/test_reductions.py [hunk 23]
```diff
@@ -62,7 +62,6 @@ def test_sum_string():
 )
 @pytest.mark.parametrize("nelem", params_sizes)
 def test_sum_decimal(dtype, nelem):
-    np.random.seed(0)
     data = [str(x) for x in gen_rand("int64", nelem) / 100]
```
Comment (wence-) [inline python/cudf/cudf/tests/test_reductions.py:65]: `gen_rand` now sets a seed?

File: python/cudf/cudf/tests/test_reshape.py [hunk 24]
```diff
@@ -35,6 +35,9 @@
     pytest_xfail = pytest.mark.skipif
 
 
+rng = np.random.default_rng(seed=0)
+
+
```
Comment (wence-) [inline python/cudf/cudf/tests/test_reshape.py:40]: *nit*: avoid global `rng` and make one in the tests?

File: python/cudf/cudf/tests/test_series.py [hunk 25]
```diff
@@ -27,6 +27,8 @@
     gen_rand,
 )
 
+rng = np.random.default_rng(seed=0)
+
```
Comment (wence-) [inline python/cudf/cudf/tests/test_series.py:32]: *nit*: avoid global rng and make one in the tests?

File: python/cudf/cudf/tests/test_series.py [hunk 26]
```diff
@@ -546,10 +548,10 @@ def test_series_datetime_value_counts(data, nulls, normalize, dropna):
 @pytest.mark.parametrize("num_elements", [10, 100, 1000])
 def test_categorical_value_counts(dropna, normalize, num_elements):
     # create categorical series
-    np.random.seed(12)
+    rng = np.random.default_rng(seed=0)
```
Comment (wence-) [inline python/cudf/cudf/tests/test_series.py:551]: Change of seed?

File: python/cudf/cudf/tests/test_stats.py [hunk 27]
```diff
@@ -19,13 +19,15 @@
 
 interpolation_methods = ["linear", "lower", "higher", "midpoint", "nearest"]
 
+rng = np.random.default_rng(seed=0)
+
```
Comment (wence-) [inline python/cudf/cudf/tests/test_stats.py:23]: *nit*: avoid global rng object?

File: python/cudf/cudf/tests/test_string.py [hunk 28]
```diff
@@ -36,6 +36,7 @@
 idx_list = [None, [10, 11, 12, 13, 14]]
 
 idx_id_list = ["None_index", "Set_index"]
+rng = np.random.default_rng(seed=0)
```
Comment (wence-) [inline python/cudf/cudf/tests/test_string.py:39]: *nit*: avoid global rng object, although I see (and I guess this occurs throughout) this is for generating values for the parametrisations, so it can't obviously live inside a test.

File: python/cudf/cudf/utils/hash_vocab_utils.py [hunk 29]
```diff
@@ -108,11 +108,11 @@ def _find_hash_for_internal(hash_bin):
             return bins, a, b
 
 
-def _perfect_hash(integers, max_constant):
+def _perfect_hash(integers, rng, max_constant):
```
Comment (wence-) [inline python/cudf/cudf/utils/hash_vocab_utils.py:111]: *nit*: Why not the last argument as for most of the other functions that are now accepting an rng?

File: python/cudf/cudf_pandas_tests/test_cudf_pandas.py [hunk 30]
```diff
@@ -1142,8 +1142,8 @@ def test_private_method_result_wrapped():
 
 
 def test_numpy_var():
-    np.random.seed(42)
-    data = np.random.rand(1000)
+    rng = np.random.default_rng(seed=0)
```
Comment (wence-) [inline python/cudf/cudf_pandas_tests/test_cudf_pandas.py:1145]: Change of seed?

File: python/cudf/cudf_pandas_tests/test_profiler.py [hunk 31]
```diff
@@ -23,12 +23,12 @@
     reason="function names change across versions of pandas, so making sure it only runs on latest version of pandas",
 )
 def test_profiler():
-    np.random.seed(42)
+    rng = np.random.default_rng(seed=0)
```
Comment (wence-) [inline python/cudf/cudf_pandas_tests/test_profiler.py:26]: Change of seed?

File: python/cudf/cudf_pandas_tests/third_party_integration_tests/tests/test_stumpy_distributed.py [hunk 32]
```diff
@@ -31,17 +31,17 @@ def dask_client():
 
 
 def test_1d_distributed(dask_client):
-    np.random.seed(42)
-    ts = pd.Series(np.random.rand(100))
+    rng = np.random.default_rng(seed=0)
```
Comment (wence-) [inline python/cudf/cudf_pandas_tests/third_party_integration_tests/tests/test_stumpy_distributed.py:34]: Change of seed?

File: python/cudf/cudf_pandas_tests/third_party_integration_tests/tests/test_stumpy_distributed.py [hunk 33]
```diff
@@ -31,17 +31,17 @@ def dask_client():
...
+    ts = pd.Series(rng.random(100))
     m = 10
     return stumpy.stumped(dask_client, ts, m)
 
 
 def test_multidimensional_distributed_timeseries(dask_client):
-    np.random.seed(42)
+    rng = np.random.default_rng(seed=0)
```
Comment (wence-) [inline python/cudf/cudf_pandas_tests/third_party_integration_tests/tests/test_stumpy_distributed.py:41]: Change of seed?

**Review by wence- — APPROVED** [review]
Thanks Prem! I had a few (likely nonblocking) comments, and I _think_ I spotted a few places where the rng is still unseeded in tests. Overall this looks good though, thanks!

File: python/cudf/cudf/tests/test_binops.py (hunk 5 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_binops.py:647]: Done.

File: python/cudf/cudf/tests/test_binops.py (hunk 4 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_binops.py:591]: Done.

File: python/cudf/cudf/tests/test_categorical.py (hunk 6 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_categorical.py:255]: It didn't actually matter keeping the seed same either:

```python
In [1]: import numpy as np

In [2]: np.random.seed(0)

In [3]: np.random.randint(0, 10)
Out[3]: 5

In [4]: np.random.default_rng(0).integers(0, 10).item()
Out[4]: 8
```

File: python/cudf/cudf/tests/test_column.py (hunk 7 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_column.py:34]: Nope, I think we must seed it. Updated.

File: python/cudf/cudf/tests/test_dataframe.py (hunk 8 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_dataframe.py:68]: Dropped it.

File: python/cudf/cudf/tests/test_dataframe.py (hunk 9 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_dataframe.py:434]: Removed the global one, the global one was only for the parametrization parts though.

File: python/cudf/cudf/tests/test_dataframe.py (hunk 10 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_dataframe.py:2852]: Done.

File: python/cudf/cudf/tests/test_dataframe.py (hunk 11 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_dataframe.py:3903]: Same reason as above, it didn't preserve the values we were previously getting/testing.

File: python/cudf/cudf/tests/test_index.py (hunk 17 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_index.py:2654]: Done in this pr itself.

File: python/cudf/cudf/tests/test_index.py (hunk 18 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_index.py:2836]: Made this change in this PR.

File: python/cudf/cudf/tests/test_groupby.py (hunk 16 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_groupby.py:1326]: Reverted to original seed.

File: python/cudf/cudf/tests/test_orc.py (hunk 19 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_orc.py:684]: yes, seed is now a pass-through param via `gen_rand_series`.

File: python/cudf/cudf/tests/test_orc.py (hunk 20 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_orc.py:989]: Fixed it.

File: python/cudf/cudf/tests/test_rank.py (hunk 22 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_rank.py:132]: Fixed it in this PR itself.

File: python/cudf/cudf/tests/test_reshape.py (hunk 24 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_reshape.py:40]: Fixed it.

File: python/cudf/cudf/tests/test_series.py (hunk 25 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_series.py:32]: Fixed.

File: python/cudf/cudf/tests/test_series.py (hunk 26 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_series.py:551]: Fixed it.

File: python/cudf/cudf/tests/test_string.py (hunk 28 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_string.py:39]: Avoiding using globals now.

File: python/cudf/cudf_pandas_tests/third_party_integration_tests/tests/test_stumpy_distributed.py (hunk 33 above)
Comment (galipremsagar) [reply python/cudf/cudf_pandas_tests/third_party_integration_tests/tests/test_stumpy_distributed.py]: Fixed.

File: python/cudf/cudf_pandas_tests/third_party_integration_tests/tests/test_stumpy_distributed.py (hunk 32 above)
Comment (galipremsagar) [reply python/cudf/cudf_pandas_tests/third_party_integration_tests/tests/test_stumpy_distributed.py]: Fixed.

File: python/cudf/cudf_pandas_tests/test_profiler.py (hunk 31 above)
Comment (galipremsagar) [reply python/cudf/cudf_pandas_tests/test_profiler.py]: Fixed.

File: python/cudf/cudf_pandas_tests/test_cudf_pandas.py (hunk 30 above)
Comment (galipremsagar) [reply python/cudf/cudf_pandas_tests/test_cudf_pandas.py]: Fixed.

File: python/cudf/cudf/tests/test_orc.py (hunk 21 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_orc.py]: Fixed it.

File: python/cudf/cudf/tests/test_dataframe.py (hunk 15 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_dataframe.py]: Fixed it.

File: python/cudf/cudf/tests/test_dataframe.py (hunk 14 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_dataframe.py]: Fixed it.

File: python/cudf/cudf/tests/test_dataframe.py (hunk 13 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_dataframe.py]: Fixed.

File: python/cudf/cudf/tests/test_dataframe.py (hunk 12 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_dataframe.py]: Fixed it.

File: python/cudf/cudf/tests/test_dataframe.py (hunk 11 above)
Comment (galipremsagar) [reply python/cudf/cudf/tests/test_dataframe.py]: Reverted to the original seed value.

File: python/cudf/cudf/testing/dataset_generator.py (hunk 3 above)
Comment (galipremsagar) [reply python/cudf/cudf/testing/dataset_generator.py]: Fixed this.

**Comment by galipremsagar** [comment]
@charlesbluca @wence- I addressed all your reviews. This should be ready for review now.

**Review by quasiben — APPROVED** [review]

File: python/dask_cudf/dask_cudf/tests/utils.py [hunk 34]
```diff
@@ -19,8 +19,9 @@
 
 
 def _make_random_frame(nelem, npartitions=2, include_na=False):
+    rng = np.random.default_rng(seed=None)
```
Comment (mroeschke) [inline python/dask_cudf/dask_cudf/tests/utils.py:22]: I still see a few `seed=None` usages which would make some test non-deterministic. Are these `None`s intentional or would we want to seed these too?

File: .pre-commit-config.yaml [hunk 35]
```diff
@@ -95,6 +95,16 @@ repos:
...
+        name: no-unseeded-default-rng
+        description: 'Enforce that no non-seeded default_rng is used and default_rng is used instead of np.random.seed'
+        entry: |
+          # Check for usage of default_rng without seeding
+          default_rng\(\)|
+          # Check for usage of np.random.seed
+          np.random.seed\(
```
Comment (mroeschke) [inline .pre-commit-config.yaml:105]: This would be covered with the `NPY002` ruff check, but not a big deal

File: python/dask_cudf/dask_cudf/tests/utils.py (hunk 34 above)
Comment (vyasr) [reply python/dask_cudf/dask_cudf/tests/utils.py:22]: FYI @galipremsagar
